namerd/storage/consul/src/main/scala/io/buoyant/namerd/storage/consul/ConsulDtabStore.scala

```diff
@@ -1,13 +1,15 @@
 package io.buoyant.namerd
 package storage.consul
 
+import java.util.regex.Pattern
```

Comment:
tioli, but can we use the scala stdlib's regex stuff instead?

Reply:
I can do that, wasn't aware scala had its own regex lib